app/models/supported_ssg.rb

```diff
@@ -69,6 +69,14 @@ def by_os_major
       all.group_by(&:os_major_version)
     end
 
+    def sorted_ssgs_for_os_major(os_major_version)
+      os_major_version = os_major_version.to_s
+      ::SupportedSsg.all
```

Comment:
```suggestion
      all
```
Also you can move this into the private block

Reply:
I moved part of this method into the private block but left the sorting part out of the private block because I needed to call it from the Profile model. Does that sound right?